Command to load test data for users and audit submissions

neilmb opened the pull request:
# Load fixture data for users and audit submissions

This adds a Django admin command `load_fixtures` that loads a list of users, specified by their usernames (Login.gov UUIDs) and then for each user it loads up a `SingleAuditChecklist` with a completely fake, but valid `general_information` form.

![Screenshot 2023-06-22 at 2 01 59 PM](https://github.com/GSA-TTS/FAC/assets/443389/6f952696-f951-4790-8368-855c8c4b92e9)

This is only the beginning of a sequence of related work, preloading submissions in various states and levels of completeness and validity.  See #1308 for more on what comes next.

This is also my first PR on this project, so I am happy to have copious comments and suggestions as to the way things are done around here.

tadhg-ohiggins commented:
I got a little carried away and created #1329 to add email addresses; that PR should also fix the linting/testing failures.

timoballard reviewed (approved):
Approving what's here in case we decide to tackle email-based user fixtures as a separate PR

tadhg-ohiggins commented:
Converting to draft for the moment in case I figure out in a short amount of time some of the improved email address approach discussed [in Slack](https://app.slack.com/client/T025AQGAN/C03QCP4FSN8/thread/C03QCP4FSN8-1687533717.860989)

neilmb commented:
@asteel-gsa I changed the test workflow back to using `docker-compose.yml` rather than `docker-compose-web.yml` because pulling a pre-built `web` image can't work with dependency changes. Can you review what I changed here to see if it will work?

asteel-gsa commented:
> @asteel-gsa I changed the test workflow back to using `docker-compose.yml` rather than `docker-compose-web.yml` because pulling a pre-built `web` image can't work with dependency changes. Can you review what I changed here to see if it will work?

Responded in slack, let me checkout this branch and make a change, test and see if it works.

asteel-gsa commented:
successfully captured faker in build step

```
Collecting faker==18.11.1 (from -r /tmp/dev-requirements.txt (line 140))
  Downloading Faker-18.11.1-py3-none-any.whl (1.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.7/1.7 MB 116.3 MB/s eta 0:00:00
```

asteel-gsa reviewed (approved):
Added build container in pr-triggers.yml on main only to catch changes to pip/requirements/dockerfile at the time of PR creation, to ensure that the conatiner on ghcr reflects local changes and does not fail when testing
